tests/robots/test_robot.py

```diff
@@ -13,54 +18,160 @@
 
 @pytest.fixture
 def panda_srdf():
-    return os.path.join(BASE_FOLDER, 'fixtures', 'panda_semantics.srdf')
+    return os.path.join(BASE_FOLDER, "fixtures", "panda_semantics.srdf")
 
 
 @pytest.fixture
 def panda_urdf():
-    return os.path.join(BASE_FOLDER, 'fixtures', 'panda.urdf')
+    return os.path.join(BASE_FOLDER, "fixtures", "panda.urdf")
 
 
 @pytest.fixture
-def panda_joints(panda_urdf, panda_srdf):
+def panda_robot_instance(panda_urdf, panda_srdf):
     model = RobotModel.from_urdf_file(panda_urdf)
     semantics = RobotSemantics.from_srdf_file(panda_srdf, model)
-    return semantics.get_configurable_joints()
+    return Robot(model, semantics=semantics)
+
+
+@pytest.fixture
+def panda_joints(panda_robot_class):
```

Comment:
also here, where is `panda_robot_class` coming from? Are you using a feature of fixtures that I'm not understanding?

Reply:
I think that's just a typo, `panda_robot_class` should be `panda_robot_instance`. But it highlights another issue, this `panda_joints` is unused so I'll remove it (and any other unused fixtures).